### DISCUSSION TITLE
Anyone able to get this working on Opensuse TumbleWeed?

### DISCUSSION THREAD

**Discussion by dusk556** [pystardust/ani-cli#875] [General] [answered]
I am a noob with Opensuse and would like to have an install command like Fedora, but for Opensuse. Thanks!

**Comment by Derisis13** [comment]
I made an Opensuse tumbleweed repo in copr. You can go ahead and use it now.
I'm a complete stranger to suse, but if I read [the docs](https://en.opensuse.org/Portal:Zypper) correctly you need to run `zypper ar https://download.copr.fedorainfracloud.org/results/derisis13/ani-cli/opensuse-tumbleweed-x86_64/` (the package is NOARCH, so it should work on any architecture).

If anyone manages to get it working, please report here so I can add it to the install instructions.

**Comment by dusk556** [comment, marked as answer]
Looks like OpenSuse is missing some resources for mpv and vlc by default. They can be added with the Packman repository found here https://en.opensuse.org/Additional_package_repositories. Ani-cli only requires the "Only Essentials" repository which has 2 versions.

TumbleWeed: "sudo zypper ar -cfp 90 https://ftp.gwdg.de/pub/linux/misc/packman/suse/openSUSE_Tumbleweed/Essentials packman-essentials"

Leap: "sudo zypper ar -cfp 90 'https://ftp.gwdg.de/pub/linux/misc/packman/suse/openSUSE_Leap_$releasever/Essentials' packman-essentials"

I would change the command for adding the Ani-cli repository to "sudo zypper addrepo https://download.copr.fedorainfracloud.org/results/derisis13/ani-cli/opensuse-tumbleweed-x86_64/ Ani-cli". Adding Ani-cli to the end of the command is adding the name of the repository, without it, the command will ask for a another argument and won't install.

I would recommend an OpenSuse Leap section seperate from OpenSuse TumbleWeed to prevent confusion.

For OpenSuse TumbleWeed

 "sudo zypper ar -cfp 90 https://ftp.gwdg.de/pub/linux/misc/packman/suse/openSUSE_Tumbleweed/Essentials packman-essentials"
"sudo zypper dup --from packman-essentials --allow-vendor-change "
"sudo zypper addrepo https://download.copr.fedorainfracloud.org/results/derisis13/ani-cli/opensuse-tumbleweed-x86_64/ Ani-cli"
"sudo zypper dup"
"sudo zypper install ani-cli"

For Opensuse Leap

"sudo zypper ar -cfp 90 'https://ftp.gwdg.de/pub/linux/misc/packman/suse/openSUSE_Leap_$releasever/Essentials' packman-essentials"
"sudo zypper dup --from packman-essentials --allow-vendor-change"
"sudo zypper addrepo https://download.copr.fedorainfracloud.org/results/derisis13/ani-cli/opensuse-tumbleweed-x86_64/ Ani-cli"
"sudo zypper dup"
sudo zypper install ani-cli"

**Reply by Derisis13** [reply]
Thanks, I'll probably update the readme. Just to be sure: did you try it on leap as well, or do you just assume it'll work there too? If you did try, which version? Knowing the exact state of testing would be useful if issues related to suse packages arise in the future.

**Reply by dusk556** [reply]
I did not try it on Leap, although the commands are the same for TumbleWeed and Leap. I'll spin up a Leap machine and update you tomorrow.

**Reply by dusk556** [reply]
Works on Leap 15.4 using the same commands I posted. There is a warning in both Leap and TumbleWeed. "ani-cli-3.4.0-1.suse.tw.noarch (ani-cli): Signature verification failed [4-Signatures public key is not available]", This can be ignored with "i" when prompted.
